lib/open-amp/resource_table.h

```diff
@@ -16,13 +16,20 @@ extern "C" {
 
 #if (CONFIG_OPENAMP_RSC_TABLE_NUM_RPMSG_BUFF > 0)
 
-#define VDEV_ID                 0xFF
+#define VDEV_ID 0xFF
+#ifdef CONFIG_IPM_RX_CHANNEL_ID
+#define VRING0_ID CONFIG_IPM_RX_CHANNEL_ID
+#else
 #define VRING0_ID 0 /* (master to remote) fixed to 0 for Linux compatibility */
+#endif
+#ifdef CONFIG_IPM_TX_CHANNEL_ID
+#define VRING1_ID CONFIG_IPM_TX_CHANNEL_ID
+#else
 #define VRING1_ID 1 /* (remote to master) fixed to 1 for Linux compatibility */
+#endif
 
 #define VRING_COUNT             2
 #define RPMSG_IPU_C0_FEATURES   1
-
 #define VRING_RX_ADDRESS        -1  /* allocated by Master processor */
```

Comment:
You removed this new line here. Please add it back. If you want to do cleanups create separate patches. But I think this is not intentional.